bundle/manifests/enterprise.splunk.com_clustermanagers.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.7.0
+    controller-gen.kubebuilder.io/version: v0.9.0
```

Comment:
when I generate it adds `x-kubernetes-map-type: atomic` for maps in CRD. i didnt see those changes here. also my kubebuilder version is 0.9.2